examples/networking/distributed_bridge/main.py

```diff
@@ -74,19 +85,39 @@ def start(self):
                 interface = host_info[i][0].interfaces["br%d" % j]
                 interface.wait_ip("99.1.0.0", 16, timeout=60)
                 print("VNI%d between host0 and host%d" % (10000 + j, i))
-                call(["ip", "netns", "exec", "host%d" % i,
-                      "ping", host0_br_ips[j], "-c", "3", "-i", "0.2", "-q"])
+                call(
+                    [
+                        "ip",
+                        "netns",
+                        "exec",
+                        "host%d" % i,
+                        "ping",
+                        host0_br_ips[j],
+                        "-c",
+                        "3",
+                        "-i",
+                        "0.2",
+                        "-q",
```

Comment:
Maybe we should ignore that one as well? It's really not helping making the code clearer.